### PR TITLE
add instructions for okta authentication

### PR DESCRIPTION
**Pull request by fyery-chen**
Address issue: https://github.com/rancher/docs/issues/1027

### CODE REVIEW

File: content/rancher/v2.x/en/admin-settings/authentication/okta/_index.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+	 Filter: Regex
+	 Value: .*
+	 
+	 ```
+4. Save your configuration, then click the **Sign-On** tab and select `View Setup Instruction`.
+5. Copy and restore the `IdP metadata`.
+6. Click the **Assignments** tab to assign this app to the specified people.
+
+## Configuring OKTA in Rancher
+
```
Comment (loganhz) [inline content/rancher/v2.x/en/admin-settings/authentication/okta/_index.md]: Change it to `## Prerequisites`

File: content/rancher/v2.x/en/admin-settings/authentication/okta/_index.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
+---
+title: Configuring OKTA (SAML)
+weight: 1200
+---
+_Available as of v2.2.0_
+
+If your organization uses OKTA Identity Provider (IdP) for user authentication, you can configure Rancher to allow your users to log in using their IdP credentials.
+
+## Prerequisites
+
```
Comment (loganhz) [inline content/rancher/v2.x/en/admin-settings/authentication/okta/_index.md]: Replace this section with the following

## Prerequisites

- In Okta, create a SAML Application with the settings below. See the [Okta documentation](https://developer.okta.com/standards/SAML/setting_up_a_saml_application_in_okta) for help.

     Setting | Value    
     ------------|------------
      `Single Sign on URL` | `https://[rancher server address]/v1-saml/okta/saml/acs` 
      `Audience URI (SP Entity ID)` | `https://[rancher server address]/v1-saml/okta/saml/metadata`

File: content/rancher/v2.x/en/admin-settings/authentication/okta/_index.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+1.	From the **Global** view, select **Security > Authentication** from the main menu.
+
+1.	Select **OKTA**.
+
+1.	Complete the **Configure Okta Account** form. The examples below describe how you can map OKTA attributes to fields within Rancher. All the fields must have been defined in **ATTRIBUTE STATEMENTS(OPTIONAL)** and **GROUP ATTRIBUTE STATEMENTS(OPTIONAL)** when configuring OKTA IdP server.
+
+
+    | Field                     | Description                                                                   |
+    | ------------------------- | ----------------------------------------------------------------------------- |
+    | Display Name Field        | The attribute that contains the display name of users. In our definition above it is `displayName`.                    |
```
Comment (loganhz) [inline content/rancher/v2.x/en/admin-settings/authentication/okta/_index.md]: remove `In our definition above ***`

File: content/rancher/v2.x/en/admin-settings/authentication/okta/_index.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+    | User Name Field           | The attribute that contains the user name/given name. In our definition above it is `userName`.                     |
```
Comment (loganhz) [inline content/rancher/v2.x/en/admin-settings/authentication/okta/_index.md]: remove `In our definition above ***`

File: content/rancher/v2.x/en/admin-settings/authentication/okta/_index.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+    | UID Field                 | An attribute that is unique to every user. Either it can be the same as `User Name Field` or you can define a different one according to your own request.                               |
+    | Groups Field              | Make entries for managing group memberships. In our definition above it is `groups`.                                 |
+    | Rancher API Host          | The URL for your Rancher Server.                                              |
+    | Private Key / Certificate | A key/certificate pair to create a secure shell between Rancher and your IdP. |
+    | Metadata XML              | The `metadata.xml` file that you restored from your IdP server in the step 5 above.               |
```
Comment (loganhz) [inline content/rancher/v2.x/en/admin-settings/authentication/okta/_index.md]: Remove `in the step 5 above.`

File: content/rancher/v2.x/en/admin-settings/authentication/okta/_index.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+
+## Configuring OKTA in Rancher
+
+
+1.	From the **Global** view, select **Security > Authentication** from the main menu.
+
+1.	Select **OKTA**.
+
+1.	Complete the **Configure Okta Account** form. The examples below describe how you can map OKTA attributes to fields within Rancher. All the fields must have been defined in **ATTRIBUTE STATEMENTS(OPTIONAL)** and **GROUP ATTRIBUTE STATEMENTS(OPTIONAL)** when configuring OKTA IdP server.
+
```
Comment (loganhz) [inline content/rancher/v2.x/en/admin-settings/authentication/okta/_index.md]: Remove `All the fields must have been defined in ATTRIBUTE STATEMENTS(OPTIONAL) and GROUP ATTRIBUTE STATEMENTS(OPTIONAL) when configuring OKTA IdP server.`

File: content/rancher/v2.x/en/admin-settings/authentication/okta/_index.md (hunk 2 above)
Comment (fyery-chen) [reply content/rancher/v2.x/en/admin-settings/authentication/okta/_index.md]: Updated!

File: content/rancher/v2.x/en/admin-settings/authentication/okta/_index.md (hunk 3 above)
Comment (fyery-chen) [reply content/rancher/v2.x/en/admin-settings/authentication/okta/_index.md]: Updated!

File: content/rancher/v2.x/en/admin-settings/authentication/okta/_index.md (hunk 4 above)
Comment (fyery-chen) [reply content/rancher/v2.x/en/admin-settings/authentication/okta/_index.md]: Updated!

File: content/rancher/v2.x/en/admin-settings/authentication/okta/_index.md (hunk 5 above)
Comment (fyery-chen) [reply content/rancher/v2.x/en/admin-settings/authentication/okta/_index.md]: Updated!

File: content/rancher/v2.x/en/admin-settings/authentication/okta/_index.md (hunk 6 above)
Comment (fyery-chen) [reply content/rancher/v2.x/en/admin-settings/authentication/okta/_index.md]: Updated!

**Comment by loganhz** [comment]
LGTM

**Comment by loganhz** [comment]
LGTM

**Comment by loganhz** [comment]
@fyery-chen  Please don't forcepush over this branch. I have updated this PR.

**Pull request merged**